style.css

```diff
@@ -0,0 +1,1769 @@
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+  align-items: center;
+}
+
+.event17 a h3,
+.event18 a h3 {
+  top: calc(50% - 24px / 2);
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.separator {
+  width: 279px;
+  padding: 0 24px;
+  margin-left: 0;
+  margin-top: -10px;
+}
+
+.event-skill {
+  width: 295px;
+  height: 144px;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  align-items: flex-start;
+  margin: 0;
+  font-weight: bold;
+  margin-left: -25px;
+}
+
+.contact-derek {
+  background-color: #6070ff;
+  border-radius: 50px 0 0 0;
+}
+
+.contact-derek h2 {
+  color: white;
+  margin-top: 40px;
+  text-align: center;
+}
+
+.contact-derek p {
+  color: #fff;
+  margin-left: 10px;
+  margin-right: 10px;
+  text-align: center;
+}
+
+#username {
+  margin-left: 20px;
+  margin-right: 20px;
+  width: 80%;
+}
+
+#email {
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-top: 20px;
+  width: 80%;
+}
+
+#detail {
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-top: 20px;
+  width: 80%;
+}
+
+#btn {
+  border-radius: 8px;
+  background-color: white;
+  text-align: center;
+  color: #396df2;
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-top: 20px;
+  width: 30%;
+  height: 50px;
+}
+
+hr.rounded {
+  border-top: 8px solid black;
+  width: 30%;
+  border-radius: 5px;
+}
+
+.form-button {
+  box-sizing: border-box;
+  align-items: center;
+  margin-left: 20px;
+  width: 12vw;
+  height: 9vh;
+  left: 0;
+  top: calc(50% - 48px / 2);
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background-color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  color: #396df2;
+  transition: transform 0.4s ease-in-out;
+}
+
+#consection {
+  display: flex;
+  flex-direction: column;
+  background: url(../images/contactformshapes.png);
+  background-repeat: no-repeat;
+  background-position: right;
+  background-color: #6070ff;
+  border-top-left-radius: 80px;
+  flex: none;
+  order: 5;
+  flex-grow: 0;
+  align-content: center;
+}
+
+.contact {
+  margin: 103px 24px 0 24px;
+  list-style: none;
+}
+
+#h1 {
+  font-family: 'Poppins', sans-serif;
+  color: white;
+  margin-top: 40px;
+  text-align: center;
+  list-style: none;
+}
+
+.contact h1 {
+  display: flex;
+  font-family: 'Poppins', sans-serif;
+  color: #fff;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  list-style: none;
+  justify-content: center;
+}
+
+.contact p {
+  display: flex;
+  font-family: 'Poppins', sans-serif;
+  color: #fff;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 28px;
+  justify-content: center;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  align-items: start;
+}
+
+form input {
+  background: #fff;
```

Comment:
- [ ] It would be better if you adjust the design of your about section to look exactly like the figma design :blush: Please make sure that the contact section text is aligned center as it is in the figma design. Also, make sure that the width of all your field and the margin-top and bottom of your `Get in touch` button looks like the Figma design :pray: 

- [ ] Please make sure you have the contact section background image and remove the horizontal overflow of your project on mobile to make your project responsive :pray: 

  ### Your design
  ![image](https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/92303891/c2ee3b55-92e3-44e1-9a2a-95565958a963)
  
  
  ### Figma Design
  ![image](https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/92303891/68bad794-a65c-4078-9dcc-0f9f59d3a8c6)

Reply:
Hi dear.

Thank you so much for your comments. 👍 

Actually, recommended changes mentioned before have been satisfactory.

![image](https://github.com/jovasal/Complete-Mobile-Figma-Portfolio/assets/35693330/f25c6309-24dc-434d-b7cf-0a8cfcf8d722)